### PR TITLE
added the word hi

### PR DESCRIPTION
**Pull request by MayaExists**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # PR-practice
 
 Jim added a line
-cool beans
+
+
+hi
```
Comment (lowerkase) [inline README.md:5]: Capitalize "hi" fool

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # PR-practice
 
 Jim added a line
-cool beans
+
```
Comment (je3m) [inline README.md:4]: do less newlines

File: README.md (hunk 1 above)
Comment (MayaExists) [reply README.md:5]: no

Comment (lowerkase) [reply README.md:5]: Do it

Comment (MayaExists) [reply README.md:5]: no

File: README.md (hunk 2 above)
Comment (MayaExists) [reply README.md:4]: i did it

**Review by je3m — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lowerkase) [reply README.md:5]: :(

**Pull request merged**